cowbird/services/service.py

```diff
@@ -12,11 +12,15 @@
 SERVICE_PRIORITY_PARAM = "priority"
 SERVICE_URL_PARAM = "url"
 SERVICE_WORKSPACE_DIR_PARAM = "workspace_dir"
+SERVICE_ADMIN_USER = "admin_user"
+SERVICE_ADMIN_PASSWORD = "admin_password"
 
 SERVICE_PARAMETERS = frozenset([
     SERVICE_PRIORITY_PARAM,
     SERVICE_URL_PARAM,
     SERVICE_WORKSPACE_DIR_PARAM,
+    SERVICE_ADMIN_USER,
```

Comment:
My bad here (not enough documentation), but those parameters were intended for "shared" parameters handled by the base service class. For service specific parameters like geoserver user/pwd you could fetch them directly here : https://github.com/Ouranosinc/cowbird/blob/master/cowbird/services/impl/geoserver.py#L23 from kwargs without declaring them anywhere else.